homeassistant/helpers/llm.py

```diff
@@ -316,7 +316,7 @@ async def async_get_api_instance(self, llm_context: LLMContext) -> APIInstance:
         """Return the instance of the API."""
         if llm_context.assistant:
             exposed_entities: dict | None = _get_exposed_entities(
-                self.hass, llm_context.assistant
+                self.hass, llm_context.assistant, include_state=False
```

Comment:
Am I correct in my understanding that the change affects only MCP server right now, since it looks like it's the only one using this flag?

Reply:
No, all are changed.
(1) All prompts now have no state
(2) MCP now uses the assist API only
(3) State is provided via new get_home_state tools via #140971